includes/admin/class-wc-admin-addons.php

```diff
@@ -39,6 +39,65 @@ public static function get_featured() {
 		}
 	}
 
+	/**
+	 * Build transient name
+	 *
+	 * @return string transient name
+	 */
+	public static function build_transient_name( $category, $term, $country ) {
+		$transient_name = 'wc_addons_extensions';
+		if ( $category ) {
+			$transient_name = $transient_name . '_category_' . $category;
+		}
+		if ( $term ) {
+			$transient_name = $transient_name . '_term_' . $term;
+		}
+		if ( $country ) {
+			$transient_name = $transient_name . '_country_' . $country;
+		}
+		return $transient_name;
+	}
+
+	/**
+	 * Build transient name
+	 *
+	 * @return string transient name
+	 */
+	public static function build_parameter_string( $category, $term, $country ) {
+
+		$paramters = array(
+			'category' => str_replace( '_', '-', $category ),
+			'term' => $term,
+			'country' => $country,
+		);
+
+		return '?' . http_build_query( $paramters );
+	}
+
+	/**
+	 * Call API to get extensions
+	 *
+	 * @return array of extensions
+	 */
+	public static function get_extension_data( $category, $term, $country ) {
+		$transient_name = self::build_transient_name( $category, $term, $country );
+		$parameters = self::build_parameter_string( $category, $term, $country );
+		$addons = get_transient( $transient_name );
+		if ( false === $addons ) {
+			$raw_extensions = wp_remote_get(
+				'https://woocommerce.com/wp-json/wccom-extensions/1.0/search' . $parameters,
+				array( 'sslverify' => false )
```

Comment:
Any reason sslverify false is used here? Since I believe wp 3.7 core includes certs with every release which gets queries when sslverify is true, instead of relying on the php install certs.

Reply:
Ahh, yes that was there from me testing on my local.  will remove!

Reply:
Done